_includes/nav.html

```diff
@@ -68,8 +72,8 @@ <h3>Informational Interviews</h3>
     {% endfor %}
   </ul>
 
+  <h3>2018 December Retreat</h3>
```

Comment:
These could be also made as variables along with folders, either in config or mapped to a data file. Might be some value to have the "nav" generated from a datafile based on discussions with ben on call about OKRs etc... that could be a section...?

Reply:
Great idea! Pushed a commit with this.

Reply:
Not sure if it accomodates what you had in mind for OKRs, but lemme know